packages/safe-ds-eda/src/apis/historyApi.ts

```diff
@@ -198,6 +202,9 @@ const deployResult = function (result: RunnerExecutionResultMessage) {
         tab.id = crypto.randomUUID();
         tabs.update((state) => state.concat(tab));
         currentTabIndex.set(get(tabs).indexOf(tab));
+    } else if (resultContent.type === 'table') {
+        updateTabOutdated(historyEntry);
+        throw new Error('Not implemented');
```

Comment:
What's the planned behavior in this case?

Reply:
@lars-reimann see #1183 where this is implemented. Essentially update column values and column types and then all other changes based on the type of action (like set respective column to sorted if it was a sort).